conrpg.cpp

```diff
@@ -437,46 +450,111 @@ class Game{
+					truenarrator("Great choice, better than \"trees\" atleast, you near the abandoned things and find the following:",50); cout << "\n\n"; truenarrator("- Tea",50); cout << "\n"; truenarrator("- An iron sword",50); cout << "\n"; truenarrator("- Golden armor with a low durability",50); cout << "\n"; truenarrator("- A shield",50); cout << "\n"; truenarrator("- Elixir of Love",50); cout << "\n"; truenarrator("- A stick",50); cout << "\n"; truenarrator("- Bandage x3",50); cout << "\n"; truenarrator("- A small backpack", 50); break;
 			}
 			else{
-				truenarrator("\t  Invalid choice. Choose again.", 50);
+				truenarrator("Invalid choice. Choose again.", 50);
 				cout << endl << endl;
 				b++;
 				continue;
 			}
 		}
 		cout << endl << endl;
-		narrator("Great.", 100); skip(250); narratorclean(" These things will come in handy. Grab them and let's get out of here.", 80); skip(1000);
+		narrator("Great.", 100); skip(250); narratorclean("\033[36m"," These things will come in handy. Grab them and let's get out of here.", 80); skip(1000);
 		cout << endl << endl;
-		cout << "      "; truenarrator("You grab the stuff and it can now be found in your inventory, which is the backpack.", 50); skip(500);
+		truenarrator("You grab the stuff and it can now be found in your inventory, which is the backpack.", 50); skip(500);
 		cout << endl << endl;
 		MCS("Good idea, let's follow that path we saw earlier, it looks like it leads to a meadow.", 80); skip(1000);
 		cout << endl << endl;
-		cout << "      "; truenarrator("You are approached by a pack of wolves ready to attack you. You have to defend yourself.", 50); cout << endl << "      "; truenarrator("In a quick manner, you have to decide which weapon to use. Your weapons: ", 50); cout << endl << "      "; truenarrator("1) A stick",50); skip(200); cout << endl << "      "; truenarrator("2) An iron sword", 50); cout << endl << endl;
-	}
-	};
-
-int main(){
-	Game game;
-	srand(time(0));
-	bool SaveExists = game.CheckSave();
-		if (SaveExists) {
-			game.SaveSequence();
+		truenarrator("You are approached by a pack of wolves ready to attack you. You have to defend yourself.", 50); cout << endl; truenarrator("In a quick manner, you have to choose the action you take: ", 50); cout << endl << endl; truenarrator("1) Use a stick to fight them",50); skip(200); cout << endl; truenarrator("2) Use an iron sword to fight them", 50); cout << endl; truenarrator("3) Try to distract them with the stick instead of fighting.", 50); cout << endl << endl;
+		cout << "\033[33m>>    ";
+		getline(cin, decision2); cout << "\033[0m" << endl;
+		if(decision2=="1"){
+			truenarrator("You decide to take out the stick and use it to fight the wolves off. Commencing combat.", 50); skip(200);
+			ambush("Wolves", 20, 3, 1, 1);
+		}
+		else if(decision2=="2"){
+			truenarrator("You decide to take out the iron sword and use it to fight the wolves off. Commencing combat.", 50); skip(200);
+			ambush("Wolves", 20, 3, 1, 1);
+		}

```

Comment:
Would be great if the code would pass arguments to the combat system for the stick and sword respectively instead of having to input them, since that would allow the player to just be OP. Should be changed ASAP.

Reply:
Yes, of course, I haven't implemented this because I need the stats of the weapons, the inventory, and an implemented level system for weapons and their stats, and player bonuses, etc. All that will be included in future versions.

Reply:
Alright, sounds good, thanks!
(Marking as resolved)